src/bindgen/find_path.cr

```diff
@@ -65,7 +65,7 @@ module Bindgen
     def find(config : PathConfig, additional = nil) : String?
       search_paths = get_search_paths(config)
       checkers = config.checks.map do |check_config|
-        Checker.create(check_config, !config.kind.directory?)
+        Checker.create(check_config, !config.kind.directory?).as(Checker)
```

Comment:
You can change the `Checker.create` method instead.  You should be fine by telling the compiler that `Checker.create` will return something of type `Checker`.  (Append ` : Checker` at the end of the `def` line of the method).  This spares you from having to write `.as` everywhere, and fits more nicely into the type system.